.github/workflows/ci.yml

```diff
@@ -198,7 +214,6 @@ jobs:
             make_targets="build-onedpl-tests"
           fi
           mkdir build && cd build
-          lscpu
```

Comment:
why are we removing `lscpu` for the `linux-testing`?

Reply:
I've reverted that change, thanks for noticing.